sdk/servicebus/azure-servicebus/samples/sync_samples/generate_sas_token_and_authenticate_client.py

```diff
@@ -48,8 +48,8 @@ def get_token(self, *scopes, **kwargs):
 
 FULLY_QUALIFIED_NAMESPACE = os.environ['SERVICE_BUS_FULLY_QUALIFIED_NAMESPACE']
 SESSION_QUEUE_NAME = os.environ["SERVICE_BUS_SESSION_QUEUE_NAME"]
-SAS_POLICY = os.environ['SERVICE_BUS_SAS_POLICY']
-SAS_KEY = os.environ['SERVICE_BUS_SAS_KEY']
+SAS_POLICY = os.environ['SERVICE_BUS_SESSION_QUEUE_SAS_POLICY']
+SAS_KEY = os.environ['SERVICE_BUS_SESSION_QUEUE_SAS_KEY']
```

Comment:
it's my fault for omitting the fact that we're using a session queue as an example.
can you help update to use a normal queue (instead of a session one) for this sample?